mirgecom/flux.py

```diff
@@ -35,6 +44,40 @@
 import numpy as np  # noqa
 
 
+def num_flux_lfr(f_minus, f_plus, q_minus, q_plus, lam, **kwargs):
+    """Lax-Friedrichs/Rusanov low-level numerical flux."""
+    return (f_minus + f_plus - lam*(q_plus - q_minus))/2
+
+
+def num_flux_central(f_minus, f_plus, **kwargs):
+    """Central low-level numerical flux."""
+    return (f_plus + f_minus)/2
+
+
+def hll_flux_driver(state_pair, physical_flux_func,
+                    s_minus, s_plus, normal):
+    """State-to-flux driver for hll numerical fluxes."""
```

Comment:
This should cite the research paper on which it is based.

Reply:
Fixed up I think.

Reply:
#648